decidim-templates/app/commands/decidim/templates/admin/update_proposal_answer_template.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Templates
+    module Admin
+      class UpdateProposalAnswerTemplate < Decidim::Command
+        # Initializes the command.
+        #
+        # template - The Template to update.
```

Comment:
If we're doing the indentation table thing we should do it with the first value too:
```suggestion
        # template    - The Template to update.
```